src/editor/__tests__/emph-stress.unit.test.ts

```diff
@@ -0,0 +1,317 @@
+import createCustomEditor from "../CustomEditor";
+import { Mark, BlockElementType } from "../types";
+import { increaseEmph, increaseStress } from "../helpers";
+
+describe("increaseEmph", () => {
```

Comment:
Should test the invariant that only one emphasis or stress level can be active at any time. Ditto with increaseStress.

Reply:
I've replaced this with a general "all marks are mutually exclusive" feature in the editor: https://github.com/UWmathNEWS/slugline-web/pull/68/commits/fc1654257f05447b3832c763b6026083e3ba690e